website/pages/home/img/why-nomad/flexible-workload-support.svg

```diff
@@ -1 +1 @@
-<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 70.16 70.16"><defs><linearGradient id="linear-gradient" y1="35.08" x2="70.16" y2="35.08" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="var(--nomad, #60dea9)"/><stop offset="1" stop-color="#008353"/></linearGradient></defs><title>15</title><rect width="70.16" height="70.16" rx="20" style="fill:url(#linear-gradient)"/><path d="M28.63,20.06H23.89a3.16,3.16,0,0,0-3.15,3.16V28" style="fill:none;stroke:#fff;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M49.15,28V23.22A3.16,3.16,0,0,0,46,20.06H41.26" style="fill:none;stroke:#fff;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M41.26,48.47H46a3.15,3.15,0,0,0,3.16-3.15V40.58" style="fill:none;stroke:#fff;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M20.74,40.58v4.74a3.15,3.15,0,0,0,3.15,3.15h4.74" style="fill:none;stroke:#fff;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/></svg>
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 70.16 70.16"><defs><linearGradient id="linear-gradient" y1="35.08" x2="70.16" y2="35.08" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="var(--nomad, #60dea9)"/><stop offset="1" stop-color="#60dea9"/></linearGradient></defs><title>15</title><rect width="70.16" height="70.16" rx="20" style="fill:url(#linear-gradient)"/><path d="M28.63,20.06H23.89a3.16,3.16,0,0,0-3.15,3.16V28" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M49.15,28V23.22A3.16,3.16,0,0,0,46,20.06H41.26" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M41.26,48.47H46a3.15,3.15,0,0,0,3.16-3.15V40.58" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/><path d="M20.74,40.58v4.74a3.15,3.15,0,0,0,3.15,3.15h4.74" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/></svg>
```

Comment:
Making this comment so I can link to it in [the SVG inline styling cleanup task in Asana](https://app.asana.com/0/1100423001970639/1200120445315586/f).